data-prepper-expression/build.gradle

```diff
@@ -27,6 +27,7 @@ dependencies {
     implementation platform('org.apache.logging.log4j:log4j-bom:2.20.0')
     implementation 'org.apache.logging.log4j:log4j-core'
     implementation 'org.apache.logging.log4j:log4j-slf4j2-impl'
+    implementation 'com.github.seancfoley:ipaddress:5.4.0'
```

Comment:
Is it OK to use such private packages in our code?

Reply:
The package is under Apache 2.0 license and actively maintained (https://github.com/seancfoley/IPAddress). It is used by many including Apache Cassandra ([ref](https://mvnrepository.com/artifact/com.github.seancfoley/ipaddress/usages)). So seems OK to me.

But if we have guidelines on thirty-party package usage that says otherwise I will follow.